R/fetch-token.R

```diff
@@ -11,7 +11,7 @@ fetch_token <- function(scopes, ...) {
     # TODO(craigcitro): Expose error handling and/or silencing here.
     try({
       token <- f(scopes, ...)
-    })
+    }, silent = TRUE)
     if (!is.null(token)) {
```

Comment:
Also made the switch to `tryCatch()` in b1466597d608eb45a0e9002c9721eb5f903fb807